docs/content/tutorials/nginx-ingress-osm.md

```diff
@@ -0,0 +1,446 @@
+## This document outlines how to integrate F5 NGINX Ingress Controller with Open Service Mesh (OSM)
```

Comment:
```suggestion
---
title: Using NGINX Ingress Controller with Open Service Mesh
description: This document explains how to integrate F5 NGINX Ingress Controller and Open Service Mesh
weight: 1800
doctypes: ["concept"]
toc: true
docs: "DOCS-1181"
---

## This document outlines how to integrate F5 NGINX Ingress Controller with Open Service Mesh
```

Without the Hugo frontmatter, this document will not be displayed.

Reply:
```
---
title: NGINX Ingress Controller and Open Service Mesh
description: |
  Use NGINX Ingress Controller with Open Service Mesh.
weight: 1800
doctypes: ["concept"]
toc: true
docs: "DOCS-1181"
---
```

minor change to Alan's suggestion to keep in line with Istio page